lib/libs/addonsettings.py

```diff
@@ -92,6 +96,17 @@ def update_settings(self):
             if not key and projectkeys:
                 self.apiconfig[provider]['apikey'] = get_projectkey(provider)
             pykodi.set_log_scrubstring(provider + '-apikey', key)
+        
+        self.pathexclusion = []
+        for index in range(2):
```

Comment:
`range(10)` to cover all 10 options added in **settings.xml**.

Reply:
Aah, some leftover from testing, nice catch !